crates/tauri-cli/src/interface/rust.rs

```diff
@@ -912,12 +911,9 @@ impl AppSettings for RustAppSettings {
       for entry in fs_bins {
         let path = entry?.path();
         if let Some(name) = path.file_stem() {
-          let bin_exists = binaries.iter().any(|bin| {
-            bin.name() == name || path.ends_with(bin.src_path().unwrap_or(&"".to_string()))
```

Comment:
this check is here to fix the following issue:
- you have a `src-tauri/src/bin/app.rs` file (automatically discovered as a binary by Cargo)
- you configure the binary manually in Cargo.toml (yeah it's silly, but it's possible and allows you to configure more stuff)
  ```toml
  [[bin]]
name = "lucass"
path = "src/bin/app.rs"
  ```

then the Tauri CLI will try to load both `app` and `lucass`, while only `lucass` exists

Reply:
reverted